src/app/services/webhooks.service.ts

```diff
@@ -158,33 +175,41 @@ const getConsoleMessage = (title, params) => {
 }
 
 // Formats webhook success info into an object to update Submissions collection
-const getSuccessDbUpdate = (response, { webhookUrl, signature }) => {
-  return { webhookUrl, signature, response: getFormattedResponse(response) }
+const getSuccessDbUpdate = (
+  response: AxiosResponse,
+  { webhookUrl, signature }: WebhookParams,
+): IWebhookResponse => {
+  return { webhookUrl, signature, ...getFormattedResponse(response) }
 }
 
 // Formats webhook failure info into an object to update Submissions collection
-const getFailureDbUpdate = (error, { webhookUrl, signature }) => {
-  const errorMessage = _.get(error, 'message')
-  const update = { webhookUrl, signature, errorMessage }
+const getFailureDbUpdate = (
+  error: Error | AxiosError,
+  { webhookUrl, signature }: WebhookParams,
+): IWebhookResponse => {
+  const errorMessage = get(error, 'message')
+  let update: IWebhookResponse = {
+    webhookUrl,
+    signature,
+    errorMessage,
+  }
   if (!(error instanceof WebhookValidationError)) {
-    update.response = getFormattedResponse(_.get(error, 'response'))
+    const { response } = getFormattedResponse(get(error, 'response'))
```

Comment:
This doesn't look like the old behaviour, it's getting `error.response.response`, is that correct?

Reply:
I changed the output of `getFormattedResponse` so that I could easily type the output as `Pick<IWebhookResponse, 'response'>`. This changed as a result.